src/wheatley.ts

```diff
@@ -331,6 +335,30 @@ export class Wheatley {
         this.setup(config).catch(this.critical_error.bind(this));
     }
 
+    private *locate_components(config: core_config) {
+        const visited = config.components?.include ? new Set<string>() : undefined;
```

Comment:
It would be a little simpler to just always create this set (this isn't something where performance matters and a set isn't expensive anyway)

Reply:
can do